### PR TITLE
demos: minor changes to operation log demo

### PR DESCRIPTION
**Pull request by martinvonz**
# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [x] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Comment by martinvonz** [comment]
I incorporated some of your ideas from #1254 here, @ilyagr. PTAL

File: demos/demo_operation_log.sh [hunk 1]
```diff
@@ -25,24 +25,27 @@ run_command "jj describe -m \"other stuff\""
 
 comment "The repo now looks like this:"
 run_command "jj log"
-comment "And the operation log looks like this:"
+comment "The most portion of the operation log is:"
```
Comment (ilyagr) [inline demos/demo_operation_log.sh:28]: This is missing the word "recent"

**Review by ilyagr — APPROVED** [review]
You might have to take another screenshot because of a typo :(

Other than that, it looks good!

File: demos/demo_operation_log.sh [hunk 2]
```diff
@@ -25,24 +25,27 @@ run_command "jj describe -m \"other stuff\""
...
+comment "Note that only the rebase was undone, and the
+subsequent \"other stuff\" change was not undone:"
 run_command "jj log"
 
 comment "We can also see what the repo looked like
 after the rebase operation:"
 run_command "jj --at-op $rebase_op log"
 
-comment "Looks nice, let's go back to that point:"
+comment "Let's say we instead want to go back to the
```
Comment (ilyagr) [inline demos/demo_operation_log.sh:43]: This change is probably be unnecessary unless you also remove the --at-op.

Comment (martinvonz) [reply demos/demo_operation_log.sh:44]: Already done and doesn't seem worse to me :)

**Comment by ilyagr** [comment]
Another tangent: if we defaulted to `relative-timestamps`, the op log lines would be a lot shorter.

If we don't want to change the default everywhere, it might make sense to have a separate `oplog-relative-timestamps` option that defaults to true. That's what a user looking at the oplog (or obslog) usually wants.

**Comment by martinvonz** [comment]
> Another tangent: if we defaulted to `relative-timestamps`, the op log lines would be a lot shorter.
> 
> If we don't want to change the default everywhere, it might make sense to have a separate `oplog-relative-timestamps` option that defaults to true. That's what a user looking at the oplog (or obslog) usually wants.

Yes, I was thinking something similar when I reviewed some recent PR: we should split not use the same `ui.relative-timestamps` config both for deciding how to display the timestamps and whether to show the end timestamp or a duration. Maybe I'll send a PR to always display the duration. Then when we have templated it, users who want two timestamps can do `format_timestamp(start_time) " - " format_timeastamp(end_time)`.

**Pull request merged**